examples/restaurant_demo/README.md

```diff
@@ -1,32 +1,28 @@
 ## Restaurant Demo
 
-PyDP library version for Google's Java Differential Privacy Library example - 
-https://github.com/google/differential-privacy/tree/master/examples/java.
+PyDP library version for Google's Java Differential Privacy Library example can be found [here](https://github.com/google/differential-privacy/tree/master/examples/java).
```

Comment:
Hi, this kinda sounds confusing. 

Could you pls change this to, 

PyDP library version for [Google's Java Differential Privacy Library - Restaurant Example](https://github.com/google/differential-privacy/tree/master/examples/java).

anchor link would be less confusing and better. Please change it to the above.